projects/1.21/assets/enderdrives/enderdrives/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+  "key.categories.enderdrives": "末影驱动器",
+  "item.enderdrives.ender_storage_component_1k": "1K 末影存储组件",
+  "item.enderdrives.ender_storage_component_4k": "4K 末影存储组件",
+  "item.enderdrives.ender_storage_component_16k": "16K 末影存储组件",
+  "item.enderdrives.ender_storage_component_64k": "64K 末影存储组件",
+  "item.enderdrives.ender_storage_component_256k": "256K 末影存储组件",
+  "item.enderdrives.ender_disk_1k": "1k 末影物品存储元件",
+  "item.enderdrives.ender_disk_4k": "4k 末影物品存储元件",
+  "item.enderdrives.ender_disk_16k": "16k 末影物品存储元件",
+  "item.enderdrives.ender_disk_64k": "64k 末影物品存储元件",
+  "item.enderdrives.ender_disk_256k": "256k 末影物品存储元件",
+  "item.enderdrives.ender_disk_creative": "创造型末影物品存储元件",
```

```suggestion
  "item.enderdrives.ender_disk_creative": "创造末影物品存储元件",
```
AE2原版的叫“创造ME元件”，作者可能没跟

---

草，我之前在游戏里没搜到